### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

**Review by mltdll — APPROVED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1 +1,16 @@
-# write your code here
+from os import remove, rename, makedirs, path
+
+
+def move_file(command: str) -> None:
+    command = command.split()[1::]
+    if "/" in command:
+        file_path, file_name = command.rsplit("/", 1)
```
Comment (Nattalli) [inline app/main.py:7]: does it work fine for you? Because I have this problem:
`
AttributeError: 'list' object has no attribute 'split'
`

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (StasOkhrym) [reply app/main.py:7]: sorry, kind of typo in `command.split()[1::]`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,17 @@
...
+        makedirs(file_path)
+    else:
+        new_file_name = command_parts[-1]
+        rename(file_name, new_file_name)
+        return
+    with open(file_name, "r") as source, \
+            open(path.join(file_path, file_name), "w") as destination:
+        destination.write(source.read())
+        remove(file_name)
```
Comment (Nattalli) [inline app/main.py:17]: but it doesn't remove the file

File: app/main.py [hunk 3]
```diff
@@ -1 +1,17 @@
-# write your code here
+from os import remove, rename, makedirs, path
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    file_name = command_parts[1]
+    if "/" in command:
+        file_path, file_name = command.rsplit("/", 1)
+        makedirs(file_path)
```
Comment (Nattalli) [inline app/main.py:9]: also, it makes not correct path:
1) `mv` at the created directory name
2) new file doesn't create

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
well done!

**Pull request closed without merging**